script/tool/lib/src/analyze_command.dart

```diff
@@ -59,52 +64,55 @@ class AnalyzeCommand extends PluginCommand {
         continue;
       }
 
-      print('Found an extra analysis_options.yaml in ${file.absolute.path}.');
-      print(
-          'If this was deliberate, pass the package to the analyze command with the --$_customAnalysisFlag flag and try again.');
-      throw ToolExit(1);
+      printError(
+          'Found an extra analysis_options.yaml in ${file.absolute.path}.');
+      printError(
+          'If this was deliberate, pass the package to the analyze command '
+          'with the --$_customAnalysisFlag flag and try again.');
+      throw ToolExit(_exitBadCustomAnalysisFile);
     }
+  }
 
+  /// Ensures that the dependent packages have been fetched for all packages
+  /// that will be analyzed.
+  Future<void> _fetchAllDependencies() async {
```

Comment:
nit: s/fetchAllDependencies/runPackagesGetOnAllDependencies

Fetch to me means it should return some value.

Reply:
Done.